SyncUp.Web/src/components/PageImageBanner.vue

```diff
@@ -28,9 +28,14 @@
         </v-col>
         <v-col align="right">
           <JoinButton
+            class="mr-2"
             :is-member="isMember"
             @toggle-membership="$emit('toggleMembership')"
           />
+          <AddGroupButton
```

Comment:
Thoughts on this UI paradigm? Add next to the search bar
![image](https://github.com/user-attachments/assets/fdc2b47c-8e0a-49bc-8201-9d2c23f8aef5)

Reply:
Love it!